gradle/owasp-suppression.xml

```diff
@@ -1,25 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <suppressions xmlns="https://jeremylong.github.io/DependencyCheck/dependency-suppression.1.3.xsd">
     <!-- See https://jeremylong.github.io/DependencyCheck/general/suppression.html for examples -->
-    <suppress until="2023-02-15">
+    <suppress until="2023-03-15">
         <notes><![CDATA[
-            Temporarily suppress CVE-2022-38752. Reporting as false positive https://github.com/OSSIndex/vulns/issues/328
-          More info on the fix here https://bitbucket.org/snakeyaml/snakeyaml/issues/531/stackoverflow-oss-fuzz-47081
+            Temporarily suppress CVE-2022-1471 till the fix is released. Web3Signer only parses specified yaml files.
```

Comment:
nit: Is there a link to track it?

Reply:
good point, updated the the bitbucket link.